py/list.go

```diff
@@ -28,6 +28,19 @@ func init() {
 		return NoneType{}, nil
 	}, 0, "append(item)")
 
+	ListType.Dict["pop"] = MustNewMethod("pop", func(self Object, args Tuple) (Object, error) {
+		var index Object = Int(0)
+		listSelf := self.(*List)
+		UnpackTuple(args, nil, "pop", 0, 1, &index)
```

Comment:
please add a check of the returned error.

Reply:
@sbinet My bad. Fixed it!